Added "weathering", "entanglement process" and "entanglement of organisms in marine debris" for #1139

iwilkie opened the pull request:
Dear editors,

As detailed in #1139 we have added "weathering", "entanglement process" and "entanglement of marine life". 

I would particularly appreciate some comments on the definitions of "weathering" and "entanglement process", as I feel like these can be made clearer and more specific.

Where available we have included interpretations in Mandarin, Italian, German and Spanish, and all 3 terms have been included in the `envoPlastics` subset.

The axioms will be added at a later stage of the project.

Thank you!

cmungall commented on src/envo/envo-edit.owl:
```diff
@@ -40266,6 +40269,49 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105120> (entanglement process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.lexico.com/definition/entangle>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105120> "An environmental system process during which objects or organisms become twisted together with or caught in each other."@de)
```
note that my suggestion was to simply not have a generic "entanglement" process. You only need marine debris entanglement. I don't know what other kinds of entanglement is useful to group here and what work a generic vague 'entanglement' buys you

pbuttigieg replied:
We'll retain the superclass - there are many forms of entanglement which we'll get to soon.

iwilkie commented on src/envo/envo-edit.owl:
```diff
@@ -40266,6 +40269,49 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105119> (weathering)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1023/A:1021148205366>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105119> "A material transformation process which produces change on a substrate due to outdoor exposure."@en)
```
> It seems that weathering is restricted to atmospheric processes (including chemical, physical, and biological agents), but please look for counterexamples.

@pbuttigieg I looked into it, so far I have not found counterexamples. I did notice that the definition is missing another aspect which shows up repeatedly, and that is that weathering occurs in situ. This distinguishes it from erosion which involves the transport of materials away from their original site ([Wikipedia](https://en.wikipedia.org/wiki/Weathering), [Britannica](https://www.britannica.com/science/weathering-geology)). What do you think of:

> A material transformation process during which the properties of materials are changed as a result of exposure to atmospheric processes while at the material's original site.

pbuttigieg replied:
@iwilkie I don't think the in situ addition is needed in the def (you can add it to the comment). I saw that condition in the sources and that's why the definition I proposed notes that the properties of the materials are changed, not their mass. 

I shouldn't have used plurals though, see below.

If you feel that it's good to note it, you can try:

> A material transformation process during which the properties of a material are changed as a result of in situ exposure to atmospheric processes.

And then add your comment explaining the difference to erosion.

iwilkie commented:
@pbuttigieg thank you for the suggestions.

I have changed the definition of weathering and added a comment as we discussed, moved "entanglement process" to "environmental system process" and adjusted the definition, renamed entanglement of marine debris to "entanglement of organisms in marine debris", modified its definition and moved it back under "entanglement process".

Let me know if there are other changes I can make.

raissameyer commented on src/envo/envo-edit.owl:
```diff
@@ -40370,6 +40373,50 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105120> "2021-05-07T18:43:55Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105120> "Verfangen"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105120> "Verhedderung"@de)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105120> "Verstrickung"@de)
```
Consider adding the German interpretation as `has_broad_synonym` as it can also be used in other contexts, e.g. as a figure of speech when talking about lies, etc.

raissameyer reviewed (changes requested):
Thanks for the addition @iwilkie, I've added a minor comment on the German interpretations.

iwilkie commented:
@raissameyer thank you for pointing that out. I now have the German interpretations as `has_broad_synonym` and noticed that the Spanish and Italian interpretations were just for "entanglement" and not "entanglement process" so fixed those.

iwilkie commented:
@raissameyer Thank you for the suggestions. I have fixed the comment of "entanglement of organisms in marine debris" as you had indicated, and changed the comment for "weathering" to reflect the definition of "erosion" in ENVO.

pbuttigieg reviewed (approved):
Looks good, merging soon

raissameyer reviewed (approved):
Many thanks @iwilkie. I'll merge
